hawkbit-ddi-resource/src/main/java/org/eclipse/hawkbit/ddi/rest/resource/DdiRootController.java

```diff
@@ -326,7 +344,10 @@ private ActionStatusCreate generateUpdateStatus(final DdiActionFeedback feedback
         }
 
         if (feedback.getStatus().getDetails() != null) {
-            messages.addAll(feedback.getStatus().getDetails());
+            messages.addAll(feedback.getStatus()
```

Comment:
Thanks @kaizimmerm. This works and looks much cleaner than the Criteria API. Will soon update the merge request soon.